gateway/api/serializers.py

```diff
@@ -117,7 +117,11 @@ def update(self, instance, validated_data):
         instance.artifact = validated_data.get("artifact")
         instance.author = validated_data.get("author")
         instance.image = validated_data.get("image")
-        instance.description = validated_data.get("description")
+
+        description = validated_data.get("description")
+        if description is not None:
```

Comment:
Nit: in Python
`if description:` is same as `if description is not None:` in most of the cases and it's simpler

Reply:
thanks for the fast test, @korgan00 , understood!